Cargo.toml

```diff
@@ -6,21 +6,26 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-hal = { package = "{{ mcu }}-hal", version = "{{ hal_version }}" }
-esp-backtrace = { version = "0.9.0", features = ["{{ mcu }}", "panic-handler", "exception-handler", "print-uart"] }
+{{ mcu }}-hal = "{{ hal_version }}"
+esp-backtrace = { version = "0.10.0", features = ["{{ mcu }}", "panic-handler", "exception-handler", "print-uart"] }
 {% if logging -%}
-esp-println = { version = "0.7.0", features = ["{{ mcu }}","log"] }
-log = { version = "0.4.18" }
+esp-println = { version = "0.8.0", features = ["{{ mcu }}", "log"] }
+log = { version = "0.4.20" }
 {% else -%}
-esp-println = { version = "0.7.0", features = ["{{ mcu }}"] }
+esp-println = { version = "0.8.0", features = ["{{ mcu }}"] }
 {% endif -%}
 {% if alloc -%}
 esp-alloc = { version = "0.3.0" }
 {% endif -%}
 {% if wifi -%}
-esp-wifi  = { version = "0.1.1", features = ["{{ mcu }}", "wifi"] }
-smoltcp = { version = "0.10.0", default-features=false, features = ["proto-igmp", "proto-ipv4", "socket-tcp", "socket-icmp", "socket-udp", "medium-ethernet", "proto-dhcpv4", "socket-raw", "socket-dhcpv4"] }
-embedded-svc = { version = "0.26.1", default-features = false, features = [] }
-embedded-io = "0.4.0"
-heapless = { version = "0.7.14", default-features = false }
+esp-wifi  = { version = "0.2.0", features = ["{{ mcu }}", "wifi"] }
+smoltcp = { version = "0.11.0", default-features=false, features = ["proto-igmp", "proto-ipv4", "socket-tcp", "socket-icmp", "socket-udp", "medium-ethernet", "proto-dhcpv4", "socket-raw", "socket-dhcpv4"] }
+embedded-svc = { version = "0.26.4", default-features = false, features = [] }
+embedded-io = "0.6.1"
+heapless = { version = "0.8.0", default-features = false }
```

Comment:
I'm not sure we should bump this since it's mostly used for `embedded-svc` which is still on `0.7+`

Reply:
oh sorry - we shouldn't downgrade embedded-io but heapless: https://crates.io/crates/embedded-svc/0.26.4/dependencies

Reason is, it's used in esp-wifi's (through embedded-svc) public API

Reply:
downgrading smoltcp is probably good since for some reason we forgot to bump it in esp-wifi 0.2.0 🤦‍♂️

Reply:
I think now, if I understood it correctly, dependencies should be fine.